packages/wallet/src/communication/actions.ts

```diff
@@ -133,3 +133,19 @@ export function isRelayableAction(action: WalletAction): action is RelayableActi
     action.type === 'WALLET.MULTIPLE_RELAYABLE_ACTIONS'
   );
 }
+
+export type CommonAction = CommitmentReceived | CommitmentsReceived;
+export function isCommonAction(
+  action: WalletAction,
+  path: string,
+  descriptor: string,
+): action is CommonAction {
+  return (
+    (action.type === 'WALLET.COMMON.COMMITMENTS_RECEIVED' ||
+      action.type === 'WALLET.COMMON.COMMITMENT_RECEIVED') &&
+    action.protocolLocator
```

Comment:
We might want to just encapsulate this in a function since we'll be using it in a lot of type guards.

Reply:
This is the encapsulated function that's used in all the type guards 😛